src/_includes/macros/video-player.html

```diff
@@ -0,0 +1,80 @@
+{# ==========================================================================
+
+   video-player.render()
+
+   ==========================================================================
+
+   Description:
+
+   Builds Video markup when given:
+
+   options             : An object used to customize the Video markup.
+
+   options.video.player: A String used to denote the type of video player.
+
+   options.video.id    : A String used to denote the id of the video.
+
+   options.video.url   : A String used to denote the url of the video.
+
+   options.image.url   : A String used to denote the url of the image.
+
+   options.button_pos  : A String used to denote the position of the video
+                         play button.
+
+                         Possible Values: 'center (default), bottomRight'
+
+   ========================================================================== #}
+
+{% macro render(options={}) -%}
+    {% import 'macros/share.html' as share %}
+    {% set video_player = options.video.player | default('youtube') %}
+    {% set video_id     = options.video.id     | default('') %}
+    {% set video_url    = options.video.url    | default('/') %}
+    {% set image_url    = options.image.url or
+                          'https://img.youtube.com/vi/' + video_id + '/maxresdefault.jpg'
+                          if video_player == 'youtube'
+    %}
+    {% set button_pos   = options.button_pos or 'center' %}
```

Comment:
Do we have multiple positions in the mockups? Is there a solid reason or can we reduce some complexity and code by reducing this to one position used everywhere?